src/main/kotlin/org/jitsi/jibri/capture/ffmpeg/executor/FfmpegExecutor.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.jitsi.jibri.capture.ffmpeg.executor
+
+import org.jitsi.jibri.sink.Sink
+import org.jitsi.jibri.util.MonitorableProcess
+
+/**
+ * Paramaters which will be passed to ffmpeg
+ */
+data class FfmpegExecutorParams(
+    val resolution: String = "1280x720",
+    val framerate: Int = 30,
+    val videoEncodePreset: String = "veryfast",
```

Comment:
Just curious: have we been using "veryfast" all along?

Reply:
yeah